### ISSUE TITLE
False positives in unicode-ranges with unit-no-unknown rule

### ISSUE THREAD

**Issue by PawelDmochowski** [stylelint/stylelint#6900]
### What steps are needed to reproduce the bug?

Use CSS with unicode-range (i.e. below from Google Fonts).

```css
/* latin */
@font-face {
  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA,
    U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215,
    U+FEFF, U+FFFD;
}
```

### What Stylelint configuration is needed to reproduce the bug?

```js
module.exports = {
  extends: ['stylelint-config-standard'],
};
```

### How did you run Stylelint?

https://stackblitz.com/edit/stackblitz-starters-anxycd

```shell
npm run lint
```

or generic

```shell
npx stylelint **/*.css
```

### Which version of Stylelint or dependencies are you using?

15.6.3

### What did you expect to happen?

No problems reported.

### What actually happened?

The following errors are reported (line numbers from stackblitz reproduction):

```
styles.css
 8:27  ✖  Unexpected unknown unit "FF"       unit-no-unknown
 8:56  ✖  Unexpected unknown unit "BB-02BC"  unit-no-unknown
 8:69  ✖  Unexpected unknown unit "C6"       unit-no-unknown
 8:77  ✖  Unexpected unknown unit "DA"       unit-no-unknown
 9:9   ✖  Unexpected unknown unit "DC"       unit-no-unknown
 9:23  ✖  Unexpected unknown unit "F"        unit-no-unknown
 9:38  ✖  Unexpected unknown unit "AC"       unit-no-unknown

7 problems (7 errors, 0 warnings)
```

### Does the bug relate to non-standard syntax?

No. Just plain CSS with standard stylelint config.

**Comment by romainmenke** [comment]
Hi @PawelDmochowski 
Thank you for reporting this.

It was previously reported in : https://github.com/stylelint/stylelint/issues/6891 and a fix has been applied.

**Comment by romainmenke** [comment]
@PawelDmochowski This issue is resolved with the release of `15.7.0` : https://github.com/stylelint/stylelint/blob/main/CHANGELOG.md#1570

Thank you again for reporting.
